kimchi/benches/amortization.rs

```diff
@@ -0,0 +1,33 @@
+use criterion::{black_box, criterion_group, criterion_main, BatchSize, BenchmarkId, Criterion};
```

Comment:
it'd be good to add some documentation here, I'm not sure what "amortization" stands for here. Also the `kimchi/README.md` has a benchmark section, it'd be good to add some words about this new benchmark there.

Reply:
sorry, the message didn't appear until I merged, will add docs in next pr.